src/group/components/GroupView.test.js

```diff
@@ -180,3 +198,120 @@ test('GroupView: Display data', async () => {
+          shareAccess: 'ALL',
+          shareUrl: 'https://example.com',
+          source: {
+            id: `de-0`,
+            createdAt: '2022-05-20T16:25:21.536679+00:00',
+            name: `Data Entry 0`,
+            createdBy: { id: 'user-id', firstName: 'First', lastName: 'Last' },
+            description: `Description 0`,
+            size: 3456,
+            entryType: 'FILE',
+            visualizations: { edges: [] },
+          },
+        },
+        errors: null,
+      },
+    });
+
+  await groupViewMemberBase({
+    id: 'user-id',
+    userRole: 'manager',
+    membershipStatus: 'APPROVED',
+    user: {
+      firstName: 'Member First Name',
+      lastName: 'Member Last Name',
+    },
+  });
+
+  const sharedDataRegion = within(
+    screen.getByRole('region', { name: 'Shared files and Links' })
+  );
+  expect(
+    sharedDataRegion.getByRole('heading', { name: 'Shared files and Links' })
+  ).toBeInTheDocument();
+  const entriesList = within(sharedDataRegion.getByRole('list'));
+  const items = entriesList.getAllByRole('listitem');
+  expect(items.length).toBe(6);
+
+  const shareButton = within(items[0]).getByRole('button', {
+    name: 'Share File “Data Entry 0”',
+  });
+  await act(async () => fireEvent.click(shareButton));
+
+  const dialog = screen.getByRole('dialog', {
+    name: 'Share “Data Entry 0” File',
+  });
+
+  // Access level
+  const shareAccess = within(dialog).getByRole('combobox', {
+    name: `Share access level`,
+  });
+  await act(async () => fireEvent.mouseDown(shareAccess));
+  const listbox = within(screen.getByRole('listbox'));
+  await act(async () =>
+    fireEvent.click(listbox.getByRole('option', { name: 'Anyone with link' }))
+  );
+
+  expect(terrasoApi.requestGraphQL).toHaveBeenCalledWith(
+    expect.stringContaining('mutation updateSharedResource'),
+    {
+      input: {
+        id: 'shared-resource-id-0',
+        shareAccess: 'ALL',
+      },
+    }
+  );
+
+  // Copy link
+  await act(async () =>
+    fireEvent.click(within(dialog).getByRole('button', { name: 'Copy Link' }))
+  );
+  const copyCall = navigator.clipboard.writeText.mock.calls[0];
+  expect(copyCall[0].toString()).toStrictEqual('https://example.com');
+});
+
+test('GroupView: Share link member', async () => {
+  await groupViewMemberBase();
+
+  const sharedDataRegion = within(
+    screen.getByRole('region', { name: 'Shared files and Links' })
+  );
+  expect(
+    sharedDataRegion.getByRole('heading', { name: 'Shared files and Links' })
+  ).toBeInTheDocument();
+  const entriesList = within(sharedDataRegion.getByRole('list'));
+  const items = entriesList.getAllByRole('listitem');
+
+  const shareButton = within(items[0]).getByRole('button', {
+    name: 'Share File “Data Entry 0”',
+  });
+  await act(async () => fireEvent.click(shareButton));
+
+  const dialog = screen.getByRole('dialog', {
+    name: 'Share “Data Entry 0” File',
+  });
+
+  // Access level disabled display text
```

Comment:
question: what does this mean?

Reply:
@paulschreiber that was an old comment, it is not relevant now, I will remove it